ctakesclient/__init__.py

```diff
@@ -1,6 +1,9 @@
 """Public API"""
 
 from . import typesystem
+from .typesystem import CtakesJSON, MatchText, Polarity, Span
+from .typesystem import UmlsTypeMention, UmlsConcept
```

Comment:
Convenience imports, so consumers don't have to use `ctakesclient.typesystem`? I kind of like the idea of having one canonical import location though -- if we do this, how do you feel about dropping the `from . import typesystem` then?